### ISSUE TITLE
Can’t download attachments from Concierge (If they were shared via Drag & Drop)

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#26328]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Chat with Concierge
2. Ask them to post a message with a few inline images that they drag and drop over the conversation
3. Open such an image and try to download it via the download button
## Expected Result: 
A message “Downloaded! Attachment successfully downloaded!” should appear, and the attachment should be available in my downloads
## Actual Result: 
Nothing happens

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 
**Reproducible in staging?:** needs reproduciton
**Reproducible in production?:** needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/84650510-733c-4470-aec5-388b00200ab4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @kidroca
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1692609027768429

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by puneetlath** [comment]
I was able to confirm that there does seem to be an issue with viewing attachments from Concierge. 

<img width="402" alt="image" src="https://github.com/Expensify/App/assets/1371996/fdff88d5-82d2-45c4-8365-fdfb0d065f20">

<img width="1193" alt="image" src="https://github.com/Expensify/App/assets/1371996/458288d2-161a-407b-9eaa-b1e8ddbd3dd5">

The fact that the thumbnail shows fine, but the full image doesn't, makes me think that this is probably a front-end issue.

**Comment by puneetlath** [comment]
Actually though, it looks like this is being handled in https://github.com/Expensify/App/issues/19965 so I'm going to close this out as a dupe.

**Comment by kidroca** [comment]
@puneetlath 

Thank you for pointing out issue #19965. However, after reviewing it closely, I believe that the two issues are not duplicates and should be treated separately.

While testing my recent PR, I uncovered a bug that is linked to a past regression. This new bug is distinct from the CORS issu being addressed in #19965. At the time of my proposal only the CORS issue existed. It took some time (maybe a month), before my proposal was selected and then I found and investigated the additional bug, my original proposal only accounts for fixing the CORS issue.

I had offered to fix the bug in question, but things have stalled a bit 

In the end we're on a crossroad 
- try to have the original author fix the regression 
- let me fix the issue by extending the scope of #19965
- look for proposals on the bug  (current ticket)
